go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/viamrobotics/canon
 go 1.19
 
 require (
-	github.com/docker/docker v24.0.6+incompatible
+	github.com/docker/docker v25.0.3+incompatible
```

Comment:
I got this version number from https://github.com/moby/moby/commit/0f77875220a3a8124660c4f7ee43400ce8b873af, which created `container.RemoveOptions`. Versions after this are acceptable, too. :upside_down_face: